ES/ESmy018.xml

```diff
@@ -260,19 +261,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <persName ref="PRS13125ZabanaKrestos">Zabāna Krǝstos</persName>.</note>
     </msItem>
 </msItem>
+                    </msItem>
```

Comment:
You forgot to close msItem number 1 here

Reply:
ok